source/java.txt

```diff
@@ -11,6 +11,8 @@ Java MongoDB Drivers
    :backlinks: none
    :depth: 1
 
+.. This page may be an unused duplicate of java-drivers.
```

Comment:
I don't remember if we already have the redirect we want in place for this page, but I think you can remove this comment from your PR.